### PR TITLE
Created new region for kitty boss

### PR DESCRIPTION
**Pull request by Grrmo**

### CODE REVIEW

**Review by JarnoWesthof — CHANGES_REQUESTED** [review]

File: worlds/timespinner/Regions.py [hunk 1]
```diff
@@ -49,9 +51,6 @@ def create_regions(world: MultiWorld, player: int, locations: Tuple[LocationData
         create_region(world, player, locations_per_region, location_cache, 'Space time continuum')
     ]
 
-    if __debug__:
```
[the remark this answers is not in the document]
Comment (Grrmo) [reply worlds/timespinner/Regions.py:52]: sure

File: worlds/timespinner/__init__.py [hunk 2]
```diff
@@ -18,7 +18,7 @@ class TimespinnerWorld(World):
     game = "Timespinner"
     topology_present = True
     remote_items = False
-    data_version = 7
+    data_version = 8
```
[the remark this answers is not in the document]
Comment (Grrmo) [reply worlds/timespinner/__init__.py:21]: Adding a region doesn't count? o_O

File: worlds/timespinner/Regions.py [hunk 3]
```diff
@@ -63,11 +62,14 @@ def create_regions(world: MultiWorld, player: int, locations: Tuple[LocationData
     connect(world, player, names, 'Lake desolation', 'Skeleton Shaft', lambda state: state._timespinner_has_doublejump(world, player))
     connect(world, player, names, 'Lake desolation', 'Space time continuum', lambda state: state.has('Twin Pyramid Key', player))
     connect(world, player, names, 'Upper lake desolation', 'Lake desolation')
-    connect(world, player, names, 'Upper lake desolation', 'Lower lake desolation') 
+    connect(world, player, names, 'Upper lake desolation', 'Eastern lake desolation')
     connect(world, player, names, 'Lower lake desolation', 'Lake desolation') 
-    connect(world, player, names, 'Lower lake desolation', 'Library') 
+    connect(world, player, names, 'Lower lake desolation', 'Eastern lake desolation')
     connect(world, player, names, 'Lower lake desolation', 'Space time continuum', lambda state: state.has('Twin Pyramid Key', player))
```
[the remark this answers is not in the document]
Comment (Grrmo) [reply worlds/timespinner/Regions.py]: .done

File: worlds/timespinner/Regions.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Grrmo) [reply worlds/timespinner/Regions.py:52]: .done

File: worlds/timespinner/__init__.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Grrmo) [reply worlds/timespinner/__init__.py]: .done

File: worlds/timespinner/Locations.py [hunk 4]
```diff
@@ -4,26 +4,28 @@
...
         LocationData('Tutorial', 'Yo Momma 1',  1337000),
         LocationData('Tutorial', 'Yo Momma 2',  1337001),
         LocationData('Lake desolation', 'Starter chest 2',  1337002),
         LocationData('Lake desolation', 'Starter chest 3',  1337003),
         LocationData('Lake desolation', 'Starter chest 1',  1337004),
         LocationData('Lake desolation', 'Timespinner Wheel room',  1337005),
-        LocationData('Upper lake desolation', 'Forget me not chest',  1337006),
-        LocationData('Lower lake desolation', 'Chicken chest',  1337007, lambda state: state._timespinner_has_timestop(world, player)),
+        LocationData('Lake desolation', 'Chicken chest', 1337007, lambda state: state._timespinner_has_timestop(world, player)),
+        LocationData('Lake desolation', 'Forget me not chest',  1337006, lambda state: state._timespinner_has_fire(world, player) and state.can_reach('Upper Lake Serene', 'Region', player)),
```
[the remark this answers is not in the document]
Comment (Grrmo) [reply worlds/timespinner/Locations.py:27]: .done

File: worlds/timespinner/__init__.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (JarnoWesthof) [reply worlds/timespinner/__init__.py]: Nope regions dont count, only location and item names & id's count. Regions dont have an id to string mapping, infect they dont even have an id, they are handled differently

**Review by JarnoWesthof — APPROVED** [review]

**Pull request merged**